doc/init.md

```diff
@@ -20,9 +20,9 @@ The macOS configuration assumes bitcoind will be set up for the current user.
 Configuration
 ---------------------------------
 
-At a bare minimum, bitcoind requires that the rpcpassword setting be set
-when running as a daemon.  If the configuration file does not exist or this
-setting is not set, bitcoind will shut down promptly after startup.
+Running bitcoind as a daemon does not require any manual configuration. You may
+set the `rpcauth` setting in the `bitcoin.conf` configuration file to override
```

Comment:
nit (feel free to ignore):

```suggestion
set the [`rpcauth` setting](/share/rpcauth) in the `bitcoin.conf` configuration file to override
```

Reply:
nit: to be clear, the `-rpcauth` option does _not_ override using a special cookie file: they work simultaneously.
See #15989.